view/script/trial.js

```diff
@@ -17,6 +17,20 @@
         }]
     };
     var ctx = document.getElementById('trialChart').getContext('2d');
+    var differenceValue = 0;
+
+    document.getElementById('toDate').setAttribute('min', document.getElementById('fromDate').value);
+    $('#fromDate').on('change', function changetoDateValue () {
```

Comment:
function names should be in camel case so `changetoDateValue` would be `changeToDateValue` or `updateToDate`.